### PR TITLE
Update sv_SE app translation for 3.8.2

### LINKED ISSUE #2310: Update sv_SE app translation for 3.8.2
Hi @genesisproject2020,
We are getting ready for the 3.8.2 release. No further changes to translatable strings are currently planned for this release.

We would be happy if you updated the Jamulus software translations for **sv_SE** by **2022-02-06**.

Please
- Update your fork from `jamulussoftware/jamulus` `master` and create a working branch
- Update translations using Qt Linguist in your fork,
- Commit and push your changes and reference this Issue,
- Open a Pull Request before 2022-02-06

Suggested commit and PR message:
```
Update sv_SE app translation for 3.8.2

Fixes #2310
```

Note: Some of the displayed "untranslated" strings will actually not be new, but will rather be shown due to technical reformatting (e.g. %1). You may be able to easily update them using the "vanished" translations list.

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md).

Thanks for contributing to Jamulus!

### PR DESCRIPTION
**Pull request by softins**
Update sv_SE app translation for 3.8.2

Fixes #2310

Done by using previous translations and Google. Changes left in "unfinished" state, so they can be checked sometime by a native speaker. Lrelease will still include them in the .qm

### CODE REVIEW

**Comment by ann0see** [comment]
@softins I updated the .ts files, so this needs a rebase. I hope it's not more work for you

**Comment by softins** [comment]
> @softins I updated the .ts files, so this needs a rebase. I hope it's not more work for you

Done

File: src/res/translation/translation_sv_SE.ts [hunk 1]
```diff
@@ -3424,48 +3424,48 @@ Vi stängde av din kanal och aktiverade &apos;Tysta mig själv&apos;. Vänligen
...
         <location filename="../../serverdlg.cpp" line="152"/>
         <source>Language combo box</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Kombineringsknapp för språk</translation>
     </message>
     <message>
         <location filename="../../serverdlg.cpp" line="157"/>
         <source>Click the button to open the dialog that allows the main recording directory to be selected.  The chosen value must exist and be writeable (allow creation of sub-directories by the user Jamulus is running as).</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Klicka på knappen för att öppna dialogrutan som gör det möjligt att välja den huvudsakliga inspelningskatalogen. Den valda platsen måste finnas och vara skrivbar (tillåta skapandet av underkataloger av den användare som Jamulus körs via.).</translation>
```
Comment (ann0see) [inline src/res/translation/translation_sv_SE.ts:3437]: Are two dots ok here?

**Review by ann0see — APPROVED** [review]
Can't say much ;-)

File: src/res/translation/translation_sv_SE.ts (hunk 1 above)
Comment (softins) [reply src/res/translation/translation_sv_SE.ts:3437]: Looks strange, I agree. I'll drop the first one.

File: src/res/translation/translation_sv_SE.ts [hunk 2]
```diff
@@ -1205,37 +1205,37 @@ Vi stängde av din kanal och aktiverade &apos;Tysta mig själv&apos;. Vänligen
...
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Om buffertfördröjningsinställningarna är inaktiverade är det förbjudet för ljuddrivrutinen att ändra denna inställning inifrån %1. På Windows trycker du på knappen ASIO enhetsinställningar för att öppna panelen för drivrutinsinställningar. På Linux använder du Jack-konfigurationsverktyget för att ändra buffertstorleken.</translation>
     </message>
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="234"/>
         <source>This opens the driver settings of your sound card. Some drivers allow you to change buffer settings, others like ASIO4ALL lets you choose input or outputs of your device(s). More information can be found on jamulus.io.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Detta öppnar drivrutinsinställningarna för ditt ljudkort. Vissa drivrutiner låter dig ändra buffertinställningar, andra som ASIO4ALL låter dig välja in- eller utgångar från din enhet (er). Mer information finns på jamulus.io.</translation>
```
Comment (hoffie) [inline src/res/translation/translation_sv_SE.ts:1233]:
```suggestion
        <translation type="unfinished">Detta öppnar drivrutinsinställningarna för ditt ljudkort. Vissa drivrutiner låter dig ändra buffertinställningar, andra som ASIO4ALL låter dig välja in- eller utgångar från din enhet(er). Mer information finns på jamulus.io.</translation>
```

If this is about the plural, I think we could drop the space there, similar to English?

File: src/res/translation/translation_sv_SE.ts [hunk 3]
```diff
@@ -2611,7 +2611,7 @@ Vi stängde av din kanal och aktiverade &apos;Tysta mig själv&apos;. Vänligen
     <message>
         <location filename="../../connectdlg.cpp" line="87"/>
         <source>If you know the IP address or URL of a server, you can connect to it using the Server name/Address field. An optional port number can be added after the IP address or URL using a colon as a separator, e.g, %1. The field will also show a list of the most recently used server addresses.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Om du känner till en servers IP-adress eller URL kan du ansluta till den i fältet Servernamn/adress. Ett valfritt portnummer kan läggas till efter IP-adressen eller URL:en med ett kolon som separator, t.ex. %1. Fältet visar också en lista över de senast använda serveradresserna.</translation>
```
Comment (hoffie) [inline src/res/translation/translation_sv_SE.ts:2614]: The `URL:en` looks a bit strange?

File: src/res/translation/translation_sv_SE.ts [hunk 4]
```diff
@@ -3424,48 +3424,48 @@ Vi stängde av din kanal och aktiverade &apos;Tysta mig själv&apos;. Vänligen
...
         <location filename="../../serverdlg.cpp" line="174"/>
         <source>Custom Directory</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Anpassad katalog</translation>
     </message>
     <message>
         <location filename="../../serverdlg.cpp" line="175"/>
         <source>The custom directory is the IP address or URL of the directory server at which the server list of the connection dialog is managed.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Den anpassade katalogen är IP-adressen eller URL:en för katalogservern där serverlistan hanteras.</translation>
```
Comment (hoffie) [inline src/res/translation/translation_sv_SE.ts:3447]:
```suggestion
        <translation type="unfinished">Den anpassade katalogen är IP-adressen eller URL:en för katalogservern där serverlistan hanteras.</translation>
```

I guess there's some meaning for `URL:en` which I haven't understood yet.

**Review by hoffie — APPROVED** [review]
Thanks for stepping in and doing all that extra work! :)

File: src/res/translation/translation_sv_SE.ts (hunk 4 above)
Comment (softins) [reply src/res/translation/translation_sv_SE.ts:3447]: Looks like it's trying to put a case ending on URL, similar to the change from "IP-adress" to "IP-adressen". I have no idea what a native Swede would consider correct! I'll see if I can ask one of the Swedes in WorldJam.

Comment (softins) [reply src/res/translation/translation_sv_SE.ts:3447]: OK, "URL:en" is fine. The -en ending is Swedish for "the", so "URL:en" means "the URL" and is a common way to write it.

**Pull request merged**